docs/dev_database/common/tile_type.rst

```diff
@@ -30,10 +30,10 @@ The :term:`tile <tile>` type files are JSON files with the following shape::
             "can_invert":' "<BOOL>",
             "dst_to_src": {
                 "delay": [
-                    "<DEALY_VALUE>",
-                    "<DELAY_VALUE>",
-                    "<DELAY_VALUE>",
-                    "<DELAY_VALUE>"
+                    "<FAST_MAX>",
```

Comment:
This order is wrong? https://github.com/SymbiFlow/prjxray/blob/1d68c6f191426b03ad497fd9dadf4dec8225fbaf/prjxray/timing.py#L174

Reply:
Fixed